### ISSUE TITLE
LA Noire DX9 crash/possible fix

### ISSUE THREAD

**Issue by Boulotaur2024** [PeterTh/gedosato#13]
This is a weird one : 

When not downsampling the overlay shows up fine and the game runs flawlessly
When downsampling the game crashes right at launch. Looking at the log, the last function call is 'redirectGetBackBuffer'... (Yes we already talked about that Durante) But the funny thing is that this function is _never_ called when not downsampling. I have no clue why. Makes no sense.

So what I do is not a fix but a workaround, I disable the 'return D3D_OK;' in 'redirectGetBackBuffer' and the game downsamples truely nicely :

http://abload.de/img/screenshot_2014-07-20vkjlq.jpg

I guess you're 100% sure the 'redirectGetBackBuffer' code is ok and works for other games (I haven't seen it called often iirc so I can't tell) ? Maybe the AddRef() needs a Release() somewhere ? 

![sans titre](https://cloud.githubusercontent.com/assets/8013986/3637485/49602200-100c-11e4-8fc3-6e9c9668f6c9.png)

So to sum it up : for LA Noire the 'redirectGetBackBuffer' method is only called once at the start of the game -and only when downsampling- and crashes the game. Here is the log file : http://pastebin.com/aMxAWu6y
